llama_stack/providers/remote/vector_io/milvus/milvus.py

```diff
@@ -48,28 +47,58 @@
 
 class MilvusIndex(EmbeddingIndex):
     def __init__(
-        self, client: MilvusClient, collection_name: str, consistency_level="Strong", kvstore: KVStore | None = None
+        self,
+        client: AsyncMilvusClient,
+        collection_name: str,
+        consistency_level="Strong",
+        kvstore: KVStore | None = None,
+        parent_adapter=None,
```

Comment:
concerned about this.

Reply:
it is part of the fix of the init order in [commit](https://github.com/llamastack/llama-stack/pull/3376/commits/19f137436f6c5b2514c03c095770c32d42d70e25)

please let me know if there is a better approach :_